### PR TITLE
Closes #1565 - Update R Versions blurb in FAQ

### LINKED ISSUE #1565: Documentation:  FAQ vignette update R version section
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

In the FAQ section (https://pharmaverse.github.io/admiral/articles/faq.html) I noticed earlier it says sentences like:

`We currently work in R Version 3.6.3 `

`Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages. ` 

@zdz2101 may make sense for you to update this as you did similar for https://github.com/pharmaverse/admiral/issues/1556. FYI @bms63

### PR DESCRIPTION
**Pull request by zdz2101**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by zdz2101** [comment]
> thanks @zdz2101 - looks fine for me, but @bms63 can you double check? i don't remember hearing exactly which version to use now so want to double check it is 4.0.5

I went off the checks in the workflow
image
image

File: vignettes/faq.Rmd [hunk 1]
```diff
@@ -49,8 +49,8 @@ knitr::opts_chunk$set(
...
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We currently work in R Version 4.0.5, but that will change as we move forward with `{admiral}`. This need for a common development environment also carries over for our choice of package versions.  
+* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 4.0.5 with a CRAN package snapshot from 2021-03-31, R Version 4.1.3 with a CRAN package snapshot from 2022-03-10 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
```
Comment (bms63) [inline vignettes/faq.Rmd:53]: So we had this discussion in another vignette.  I think we should make this language more vague so we don't have to update it again.

File: vignettes/faq.Rmd [hunk 2]
```diff
@@ -49,8 +49,8 @@ knitr::opts_chunk$set(
...
-* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We currently work in R Version 4.0.5, but that will change as we move forward with `{admiral}`. This need for a common development environment also carries over for our choice of package versions.
```
Comment (bms63) [inline vignettes/faq.Rmd:52]:
```suggestion
* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We currently work in earliest of the three latest R Versions. This need for a common development environment also carries over for our choice of package versions.  
```

File: vignettes/faq.Rmd (hunk 1 above)
Comment (bms63) [inline vignettes/faq.Rmd:53]:
```suggestion
* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against the last 3 R Versions and the closest snapshots of packages to those R versions.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
```

**Review by bms63 — COMMENTED** [review]
I think these two sections should be identical honestly.  https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#r-and-package-versions-for-development.  I'm having trouble coming up with the best phrasing but I think my overall goal is to not have to update this when a new version of R comes out or new snapshots

**Comment by rossfarrugia** [comment]
makes sense to me @bms63!

**Review by bms63 — APPROVED** [review]
Love it!

**Pull request merged**